app/views/shared/_block_link.html.erb

```diff
@@ -0,0 +1,20 @@
+<%#
+yields: Link text (required).
+locals:
+* url: URL of link.
+%>
+<%= tag.a href: url, class: 'usa-link block-link' do %>
+  <%= yield %>
```

Comment:
if the yield is required, should we consider throwing an error if it's not provided?

```suggestion
  <%= yield.presence || (raise "block_link content not provided") %>
```

Reply:
> if the yield is required, should we consider throwing an error if it's not provided?

Yeah, we do this in one of the other similar partial components, so it could make sense to have here as well.

Reply:
>if the yield is required, should we consider throwing an error if it's not provided?

Updated in ea26ce860.